test/prototype/moe_training/test_kernels.py

```diff
@@ -272,10 +272,10 @@ def test_mxfp8_per_group_blocked_scales_3d(
 
 
 @skip_if_rocm("ROCm enablement in progress")
-@pytest.mark.parametrize("m", [256, 512, 1024, 5120])
-@pytest.mark.parametrize("total_k", [512, 1024, 2048, 4096, 8192, 16384])
-@pytest.mark.parametrize("n_groups", [1, 4, 8, 16])
-def test_mxfp8_per_group_blocked_scales_2d2d(
+@pytest.mark.parametrize("m", [256])
```

Comment:
Intentional?

Reply:
Nope, was for debugging this issue: https://github.com/pytorch/ao/pull/2990/files#r2354167396 - will fix before landing, thanks.